src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
+			double noise_bound = 1.0 + 0.5 + 0.25;
+			double density = noise / noise_bound * 0.5 + 0.5;
+			grid[i] = (density < m_density);
```

Comment:
No, if I recall correctly, `noise` is between -1 and 1 (the `// comment` really refers to both lines), so halving that gives -0.5 to 0.5, plus 0.5 is 0 to 1. It's not uniform, but 0.4 works out to about 40% cloud cover.